scripts/sct_run_batch.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python
+##############################################################################
+#
+# Wrapper to processing scripts, which loops across subjects. Data should be
+# organized according to the BIDS structure:
+# https://github.com/sct-pipeline/spine_generic#file-structure
+#
+# ----------------------------------------------------------------------------
+# Copyright (c) 2020 Polytechnique Montreal <www.neuro.polymtl.ca>
+# Authors: Julien Cohen-Adad, Chris Hammill
+#
+# About the license: see the file LICENSE.TXT
+##############################################################################
+
+from __future__ import absolute_import
+
+import os
+import sys
+import argparse
+import numpy as np
+import multiprocessing
+import subprocess
+import re
+import time
+
+import spinalcordtoolbox.image as msct_image
+from spinalcordtoolbox.image import Image, concat_data
+from spinalcordtoolbox.utils import Metavar, SmartFormatter
+
+parser = argparse.ArgumentParser(
+  description = 'Wrapper to processing scripts, which loops across subjects. Data should be '
+                'organized according to the BIDS structure: '
+                'https://github.com/sct-pipeline/spine_generic#file-structure.\n'
+                'The processing script (task) should accept a subject directory as its only argument. '
+                'Additional information is passed via environment variables \n'
+                'PATH_SEGMANUAL (--path-segmanual)\n'
```

Comment:
instead of listing all the shell env variables here, why not saying that the name of the env variables to create appears next to each flag (in capital letter)?

Reply:
What do you mean regarding the environment variables? I have coded it so that the subprocess inherits all environment variables from the parent process.

Reply:
@jcohenadad was there more I needed to do for this?